main.nf

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env nextflow
 
-//IRAP_CONFIGS = Channel.fromPath( "${params.irapConfigDir}/*.conf").filter{it.baseName == 'mus_musculus'}
-IRAP_CONFIGS = Channel.fromPath( "${params.irapConfigDir}/*.conf")
+IRAP_CONFIGS = Channel.fromPath( "${params.irapConfigDir}/*.conf").filter{it.baseName == 'drosophila_melanogaster'}
```

Comment:
I'm guessing that using fly here is temporary for some tests, correct? Make sure to change before merging.